src/Validation/Indisposable.php

```diff
@@ -2,52 +2,120 @@
 
 namespace Propaganistas\LaravelDisposableEmail\Validation;
 
-use Illuminate\Support\Str;
+use Illuminate\Support\Facades\Cache as FrameworkCache;
+use Illuminate\Support\Facades\Log;
+use Exception;
+
+class Indisposable {
 
-class Indisposable
-{
     /**
-     * The array of disposable domains.
+     * The remote JSON source URI.
      *
-     * @var array
+     * @var string
+     */
+    protected $sourceUrl = 'https://rawgit.com/andreis/disposable-email-domains/master/domains.json';
```

Comment:
Make it `public static` or implement a setter so the source URL can be changed when desired.